13LessonThirteen/in-class-project/index.html

```diff
@@ -1,10 +1,82 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title></title>
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+    <main>
+        <section class="row">
+            <div class="col-lg-10 col-xs-12">
+                <img class="img-responsive frontimage" src="css/bg.jpg" />
+            </div>
+            <section class="col-lg-2 visible-lg sidebar">
+                <h3>Top Stories</h3>
+                <p>
+                    Stanford Researchers Mark Artificial Skin that sense touch
+                </p>
+                <p>
+                    3 New Ways to wear your favorite chambray shirt
+                </p>
+                <p>
+                    Where to buy your next rug
+                </p>
+                <p>
+                    Chukkas that offer a casual, comfortable, style
+                </p>
+            </section>
+        </section>
+        <section class="frontpage">
+            <h1>Tomorrow's Technology, Yesterday's Industry</h1>
+            <p>
+                Sometimes what the tech world comes up with has already been tried and rejected and really doesn't need to be resurrected-like the Taylorist view...
+            </p>
+            <a href="#" class="button">TECHNOLOGY</a>
+        </section>
+        <footer class="visible-xs">
+            <section class="row">
+                <section class="col-xs-6 links">
+                    <h4>Useful Links</h4>
+                    <p>
+                        About
+                    </p>
+                    <p>
+                        Jobs
+                    </p>
+                    <p>
+                        Team
+                    </p>
+                    <p>
+                        Advertise
+                    </p>
+                    <p>
+                        Terms
+                    </p>
+                </section>
+                <section class="col-xs-6 social">
+                    <h4>Social Media</h4>
+                    <p>
+                        Facebook
+                    </p>
+                    <p>
+                        Instagram
+                    </p>
+                    <p>
+                        Twitter
+                    </p>
+                </section>
+            </section>
+        </footer>
+    </main>
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
this script get you jQuery, you are missing the one that gets you the JavaScript library for Bootstrap. Check the template I sent to the channel